kotlinx-coroutines-core/build.gradle.kts

```diff
@@ -58,6 +60,7 @@ kotlin {
                 implementation("org.openjdk.jol:jol-core:0.16")
             }
         }
+        setupBenchmarkSourceSets(this)
```

Comment:
Why is this a separate function? Simply for the sake of readability, or are there other motives?

Also, I'd prefer `setupBenchmarkSourceSets(sourceSets)` outside the `sourceSets` block, so that it's clearer what gets passed as the parameter.

Reply:
For the sake of readability, to confine benchmark configuration which is far from trivial.
If it were `benchmarks { jvm {} ...}`, I would have left it here